src/pages/PlanPage/subRoutes/UpgradePlanPage/UpgradePlanPage.jsx

```diff
@@ -41,16 +40,7 @@ function UpgradePlanPage() {
 
   return (
     <div className="flex flex-col gap-8 md:w-11/12 md:flex-row lg:w-10/12">
-      <UpgradeDetails
-        accountDetails={accountDetails}
-        plan={plan}
-        plans={plans}
-        proPlanMonth={proPlanMonth}
-        proPlanYear={proPlanYear}
-        sentryPlanMonth={sentryPlanMonth}
-        sentryPlanYear={sentryPlanYear}
-        scheduledPhase={scheduledPhase}
-      />
+      <UpgradeDetails />
```

Comment:
It's so beautiful 🥲

Reply:
It truly is, it took a while, but it's worth